src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,107 @@
+            while (resultSet.next()) {
+                manufacturer = new Manufacturer();
+                Long id = resultSet.getObject("id", Long.class);
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                manufacturer.setId(id);
+                manufacturer.setName(name);
+                manufacturer.setCountry(country);
+                manufacturerList.add(manufacturer);
+            }
+        } catch (SQLException throwables) {
+            throw new RuntimeException(" is not good connection method getAll ", throwables);
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "update Manufacturer set name = ?, country = ? where id = ?";
+        try (Connection connect = ConnectionUtil.getConnect()) {
+            PreparedStatement preparedStatement = connect.prepareStatement(query);
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setLong(3, manufacturer.getId());
+            preparedStatement.executeUpdate();
+        } catch (SQLException throwable) {
+            throw new RuntimeException(" is not good connection in method update ", throwable);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String queryDelete = "update Manufacturer set id_delete = true where id = ?";
+        try (Connection connect = ConnectionUtil.getConnect();
+        ) {
+            PreparedStatement preparedStatement = connect.prepareStatement(queryDelete);
+            preparedStatement.setLong(1, id);
+            preparedStatement.executeUpdate();
+        } catch (SQLException throwable) {
+            throw new RuntimeException("is not connection in method delete ", throwable);
+        }
+        return false;
+    }
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertInto = "insert into Manufacturer (id,name,country,id_delete) values (?,?,?,?)";
+        try (Connection connect = ConnectionUtil.getConnect()) {
+            PreparedStatement preparedStatement = connect.prepareStatement(insertInto);
+            preparedStatement.setLong(1, manufacturer.getId());
+            preparedStatement.setString(2, manufacturer.getName());
+            preparedStatement.setString(3, manufacturer.getCountry());
+            preparedStatement.setBoolean(4, false);
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("is not good connection in method create", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        Manufacturer manufacturer = new Manufacturer();
```

Comment:
```suggestion
        Manufacturer manufacturer = null;
```
if the Manufacturer is not found return a Optional.empty() not an extual instance of Manufacturer